delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/operations/remote/client/RemoteDiskStorageClient.scala

```diff
@@ -68,7 +67,7 @@ final class RemoteDiskStorageClient(client: HttpClient, getAuthToken: AuthTokenP
       val req      = Head(endpoint).withCredentials(authToken)
       client(req) {
         case resp if resp.status.isSuccess() => IO.delay(resp.discardEntityBytes()).void.toUIO
```

Comment:
Feels weird we still have this `toUIO`

Reply:
Yeah it's probably fixed in another commit I've already made 😓